app/models/payment.rb

```diff
@@ -1,43 +1,35 @@
 class Payment < ApplicationRecord
```

Comment:
@shshamim63 Can you think of ways to improve the responsibility of the `Payment` class?

Reply:
Yes, I think a separate service is a more feasible way for this task.